OloEngine/src/Platform/OpenGL/OpenGLTextureCubemap.cpp

```
@@ -60,6 +60,55 @@ namespace OloEngine
             OLO_CORE_ASSERT(false, "Unknown ImageFormat!");
             return 0;
         }
+
+        struct FormatInfo
+        {
+            u32 BytesPerPixel = 4;
+            GLenum DataType = GL_UNSIGNED_BYTE;
+        };
+
+        [[nodiscard("Store this!")]] static FormatInfo GetFormatInfo(ImageFormat format)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)